src/components/LeftSidebar/SearchBox/SearchBox.vue

```diff
@@ -25,7 +25,9 @@
 			:value.sync="localValue"
 			:label="placeholderText"
 			:show-trailing-button="isSearching"
+			:disabled="disabled"
 			trailing-button-icon="close"
+			v-on="$listeners"
 			@trailing-button-click="abortSearch"
```

Comment:
Need to check that line and method, because text doesn't cleared up then clicking on 'X'


https://github.com/nextcloud/spreed/assets/93392545/f8a7bba9-813f-4a4a-bd5e-a7b73cf3c17c

Reply:
```vue
				:value.sync="searchText"
```
Try this to synchronise both SearchBox and TextField values, and then check for abortSearch()